### PR TITLE
update

### PR DESCRIPTION
**Pull request by Sivichoudhary**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,13 +11,12 @@
   },
   "dependencies": {
     "axios": "^1.7.2",
+    "cors": "^2.8.5",
     "express": "^4.19.2",
     "firebase": "^10.12.2",
     "lodash": "^4.17.21",
     "mongodb": "^6.8.0",
     "mongoose": "^8.4.5",
-    "firebase": "^10.12.2",
```
Comment (shabdpatel) [inline package.json:19]: Why did you remove firebase from package.json? @Sivichoudhary

**Review by shabdpatel — COMMENTED** [review]
Is Add schlorship working properly?

File: package.json (hunk 1 above)
Comment (Sivichoudhary) [reply package.json:19]: sir,i did'nt this intentionally , when i update my system all these things done automatically and i did'nt pay any attention to this ,sorry for that
